### ISSUE TITLE
[BUG] (Again, incorrect display of the percentage of translation in the Ukrainian language)

### ISSUE THREAD

**Issue by sergd88** [marticliment/WingetUI#1394]
### Please confirm these before moving forward

- [X] I have searched for my issue and not found a work-in-progress/duplicate/resolved issue.
- [X] I have tested that this issue has not been fixed in the latest [(beta or stable) release](https://github.com/marticliment/WingetUI/releases/).
- [X] I have checked the [FAQ](https://github.com/marticliment/WingetUI#frequently-asked-questions) section for solutions
- [X] This issue is about a bug (if it is not, please use the [correct template](https://github.com/marticliment/WingetUI/issues/new/choose))

### Describe your issue

Again, incorrect display of the percentage of translation in the Ukrainian language.

### WingetUI Log

```text
-
```

### Package Managers Logs

```text
-
```

### Screenshots and videos

![1](https://github.com/marticliment/WingetUI/assets/125032334/1b817ebc-58bd-4c9f-a4e8-4e8947a9606d)

**Comment by marticliment** [comment]
did this happen before?

**Comment by sergd88** [comment]
> did this happen before? I was not aware of it

Yes, there was a very similar situation before, [#1101](https://github.com/marticliment/WingetUI/issues/1101), and you fixed it. In the English and Ukrainian translations, the completion percentage was different. Now it's essentially the same, just with a different percentage. The actual translation into Ukrainian according to the GitHub page and the English version of the program is 96%. If you look at the Ukrainian version of the application, when the Ukrainian translation is directly engaged, as you could see in the screenshot, it's for some reason 4% displayed. From what I can to guess, the 4% might have come from the difference between 100%, which is the full translation, and the actual Ukrainian translation at 96%. I don't know any other reason why the 4% figure might be appearing.

**Comment by marticliment** [comment]
I see the issue now. It reports properly unless the UI is in Ukranian

**Comment by sergd88** [comment]
> I see the issue now. It reports properly unless the UI itself shows in Ukranian

Yes, the issue only occurs when the Ukrainian language is directly used for the interface. This is not critical at all, but similar issues might be present in other translations, which I haven't checked, so I decided to report it to keep you informed.
